tests/nosetests/pyanaconda_tests/module_storage_test.py

```diff
@@ -73,6 +75,21 @@ def setUp(self):
         self.storage_module = StorageModule()
         self.storage_interface = StorageInterface(self.storage_module)
 
+    def _test_dbus_partitioning(self, publisher, expected_method):
```

Comment:
Please name this `_check...` to avoid misunderstanding that this is a test.

Reply:
Fixed.